src/components/ui4/RowUi4.tsx

```diff
@@ -59,7 +59,7 @@ export const RowUi4 = (props: Props) => {
       try {
         await onPress()
       } catch (err) {
-        showError(err)
+        showDevError(err)
```

Comment:
Since this is already inside `EdgeTouchableOpacity`, just delete the whole `try / catch` block.